src/crimson/os/seastore/cache.cc

```diff
@@ -613,7 +622,6 @@ void Cache::register_metrics()
 
 void Cache::add_extent(CachedExtentRef ref)
 {
-  LOG_PREFIX(Cache::add_extent);
```

Comment:
I'd sort of like to keep these, perhaps at TRACE?

Reply:
These are private methods, some are removed as they are instead logged by the caller and there are more information from the caller.

I'll revisit them, add comments if the removed log is coverred by caller, and keep the rest.

Reply:
Eh, if they're covered elsewhere that's fine.